src/tools/Reporting/Reporting/Counter.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+
+namespace Reporting
+{
+    public class Counter
+    {
+        public string Name { get; set; }
+
+        public bool TopCounter { get; set; }
+
+        public bool DefaultCounter { get; set; }
+
+        public bool HigherIsBetter { get; set; }
+
+        public string MetricName { get; set; }
+
+        public IList<double> Results { get; set; }
```

Comment:
nit: maybe `IReadOnlyList` to make sure no new elements are added?

Reply:
BDN happens to know all the results at once but you could imagine another tool which wants to add results as it parses them out (a linear parser for ETLs, maybe.)